lib/jsdom/living/nodes/HTMLTableElement.webidl

```diff
@@ -1,33 +1,37 @@
+[Exposed=Window,
+ HTMLConstructor]
 interface HTMLTableElement : HTMLElement {
-  attribute HTMLTableCaptionElement? caption;
+  [CEReactions] attribute HTMLTableCaptionElement? caption;
   HTMLTableCaptionElement createCaption();
-  void deleteCaption();
-  attribute HTMLTableSectionElement? tHead;
+  [CEReactions] void deleteCaption();
+
```

Comment:
@domenic do we need extra spacing here? I know you've mentioned this to me before.

Reply:
This is the way it looks in the [spec](https://html.spec.whatwg.org/#the-table-element). We would prefer to make as few changes to the original formatting as possible to make updates like this one easier in the future.